tests/bluetooth/tester/src/btp/btp_vocs.h

```diff
@@ -0,0 +1,22 @@
+/* btp_vocs.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/sys/util.h>
+
+#define VOCS_READ_SUPPORTED_COMMANDS	0x01
+
+#define VOCS_UPDATE_LOC			0x02
+#define VOCS_UPDATE_DESC		0x03
+
+struct vocs_audio_desc {
+	uint8_t desc[8];
```

Comment:
```suggestion
	char desc[8];
```

Reply:
I think this should be  len + uint8 value[0] in btp   (null termination excluded)

Reply:
Done